config/main.py

```diff
@@ -705,7 +705,7 @@ def load(filename, yes):
 @click.argument('filename', required=False)
 def reload(filename, yes, load_sysinfo):
     """Clear current configuration and import a previous saved config DB dump file.
-       <filename> : Names of configuration file(s) to use, separated by comma with no spaces in between
+       <filename> : Names of configuration file(s) to reload, separated by comma with no spaces in between
```

Comment:
s/reload/load/. "Reload" simply means clear the DB first; the file(s) just get loaded. "Reload" is a bit confusing terminology as it is. Sorry to nitpick.

Reply:
Thanks Joe, updated. yes the wording to use "reload" was confusing.